pkg/issuer/ca/issue.go

```diff
@@ -83,13 +83,24 @@ func (c *CA) Issue(ctx context.Context, crt *v1alpha1.Certificate) (*issuer.Issu
 		return nil, err
 	}
 
+	caCert := caCerts[0]
+
 	// sign and encode the certificate
 	certPem, _, err := pki.SignCertificate(template, caCert, signeePublicKey, caKey)
 	if err != nil {
 		c.Recorder.Eventf(crt, corev1.EventTypeWarning, "ErrorSigning", "Error signing certificate: %v", err)
 		return nil, err
 	}
 
+	// encode the chain
+	// TODO: replace caCerts with caCerts[1:]?
```

Comment:
@mikebryant can you comment on your thoughts here? From what I can tell, we'll currently put `<signed cert> <intermediate cert> <root ca>` into the `tls.crt` field in the secret. I would have thought we only want `<signed cert> <intermediate cert>`?

Reply:
`EncodeX509Chain` will ignore any self-signed (i.e. root) certificates.

so `chainPem` should end up as `<intermediate>`, which is then concatenated onto `certPem` to make `<cert> <intermediate>`